main.go

```diff
@@ -47,6 +48,7 @@ func init() {
 	flag.UintVar(&opts.containerSubnetLength, "container-subnet-length", 8, "container subnet length")
 	flag.StringVar(&opts.etcdEndpoints, "etcd-endpoints", "http://127.0.0.1:4001", "a comma-delimited list of etcd endpoints")
 	flag.StringVar(&opts.etcdPath, "etcd-path", "/registry/sdn/", "etcd path")
+	flag.StringVar(&opts.minionPath, "minion-path", "/kubernetes.io/registry/minions/", "etcd path that will be watched for minion creation/deletion (Note: -sync flag will override this path with -etcd-path)")
```

Comment:
I'm pretty sure that it should be /kubernetes.io/mionions instead, here's the output from a fresh install with one minion.

```
# curl -s -X GET --cacert /etc/openshift/master/ca.crt --cert /etc/openshift/master/master.etcd-client.crt --key /etc/openshift/master/master.etcd-client.key -X GET https://127.0.0.1:4001/v2/keys/?recursive=true -k | json_reformat  | grep minion
                        "key": "/kubernetes.io/minions",
                                "key": "/kubernetes.io/minions/ose3.os1.phx2.redhat.com",
```